examples/timit/asr1/README.md

```diff
@@ -0,0 +1,195 @@
+# Transformer/Conformer ASR with Timit
```

Comment:
seems without conformer training in this config. Also fix `examples/timit/README.md`

Reply:
roger